kolibri/utils/cli.py

```diff
@@ -12,7 +12,12 @@
 from django.core.management import execute_from_command_line
 
 import kolibri
-from kolibri.plugins import config
+
+try:
+    from kolibri.plugins import config
+except RuntimeError as e:
+    logging.error(str(e))
```

Comment:
possibly add a little more context:

```suggestion
except RuntimeError as e:
    logging.error("Could not load plugin configuration: {}".format(e))
```